images/webhook/Dockerfile

```diff
@@ -0,0 +1,30 @@
+#
```

Comment:
This is a test/mock image?

Perhaps we should have a parent folder to make that more obvious?

Reply:
done